ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -478,6 +493,47 @@ internal class CustomerCenterViewModelImpl(
         }
     }
 
+    override fun refreshStateIfLocaleChanged() {
+        val currentLocaleList = getCurrentLocaleList()
+        if (_lastLocaleList.value != currentLocaleList) {
+            _lastLocaleList.value = currentLocaleList
+        }
+    }
```

Comment:
Since this is only for tracking purposes, I assumed there's no need to do a state update

Reply:
Right, as long as we don't need to update the state it makes sense... I guess with CustomerCenter we are not updating on locale changes anyway since we need to refetch the backend response so makes sense 👍